pkg/apis/internal.admin.acorn.io/v1/baseresources_test.go

```diff
@@ -103,122 +103,115 @@ func TestAdd(t *testing.T) {
 	}
 }
 
-func TestRemove(t *testing.T) {
+func TestBaseResourcesRemove(t *testing.T) {
 	// Define test cases
 	testCases := []struct {
 		name     string
-		current  Resources
-		incoming Resources
+		current  BaseResources
+		incoming BaseResources
 		all      bool
-		expected Resources
+		expected BaseResources
 	}{
 		{
-			name:    "remove from empty resources",
-			current: Resources{},
-			incoming: Resources{
+			name:    "remove from empty BaseResources resources",
+			current: BaseResources{},
+			incoming: BaseResources{
 				Apps:   1,
 				Memory: resource.MustParse("1Mi"),
 			},
-			expected: Resources{},
+			expected: BaseResources{},
 		},
 		{
-			name: "remove from existing resources",
-			current: Resources{
+			name: "remove from existing BaseResources resources",
+			current: BaseResources{
 				Apps:   1,
 				Memory: resource.MustParse("1Mi"),
 			},
-			incoming: Resources{
+			incoming: BaseResources{
 				Apps:   1,
 				Memory: resource.MustParse("1Mi"),
 			},
-			expected: Resources{},
+			expected: BaseResources{},
 		},
 		{
 			name: "should never get negative values",
 			all:  true,
-			current: Resources{
+			current: BaseResources{
 				Apps:          1,
 				Memory:        resource.MustParse("1Mi"),
-				Secrets:       1,
 				VolumeStorage: resource.MustParse("1Mi"),
 			},
-			incoming: Resources{
+			incoming: BaseResources{
 				Apps:          2,
 				Memory:        resource.MustParse("2Mi"),
-				Secrets:       2,
 				VolumeStorage: resource.MustParse("2Mi"),
 			},
-			expected: Resources{},
+			expected: BaseResources{},
 		},
 		{
 			name: "remove persistent counts with all",
-			current: Resources{
-				Secrets:       1,
+			current: BaseResources{
 				VolumeStorage: resource.MustParse("1Mi"),
 			},
-			incoming: Resources{
-				Secrets:       1,
+			incoming: BaseResources{
 				VolumeStorage: resource.MustParse("1Mi"),
 			},
 			all:      true,
-			expected: Resources{},
+			expected: BaseResources{},
 		},
 		{
 			name: "does not remove persistent counts without all",
-			current: Resources{
-				Secrets:       1,
+			current: BaseResources{
 				VolumeStorage: resource.MustParse("1Mi"),
 			},
-			incoming: Resources{
-				Secrets:       1,
+			incoming: BaseResources{
 				VolumeStorage: resource.MustParse("1Mi"),
 			},
-			expected: Resources{
-				Secrets:       1,
+			expected: BaseResources{
```

Comment:
This test is no longer useful/needed since secrets are no longer part of it.

Reply:
Talked offline, but `VolumeStorage` is a persistent quantity, not a count. The same logic should still be applied so I think all that needs changing here is the name of the test.